coatrack/coatrack
# Feature/170 gateway deployment via docker

closes #170

## How to test this PR
* Deploy CoatRack
  * via docker-compose: `./build-and-deploy-images-locally.sh`
  * via IntelliJ Runner. Note: For this approach, you need a `coatrack/coatrack-proxy` docker image. You can create it by executing `./helper-script-building-the-docker-images-and-exporting-environment-variables.sh`.
* Do the admin tutorial and copy the Service Access URL
* Go to the list of gateways, click on the second/right download button. A docker.compose.yml should be downloaded.
* Go to downloads directory and `docker-compose up`. The bottom line of the logs should show sth like "Switching to online mode".
* Apply the Service Access URL to your browser. You should be redirected to the service defined in the CoatRack GUI.
